article/_posts/2021-07-29-DiSCo.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: review
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry diffusion
```

Comment:
Maybe "diffusion-mri" is a more descriptive/complete tag?

Reply:
Changed for dMRI in d7c2ce9eb767b433be27444ba37507eda745dfad